snooty/rstspec.toml

```diff
@@ -4,7 +4,7 @@ version = 0
 [enum]
 alignment = ["left", "center", "right"]
 backlinks = ["entry", "top", "none"]
-banner_variant = ["info", "warning", "danger"]
+banner_variant = ["info", "warning", "danger", "cta-banner"]
```

Comment:
To clarify, this permits:
```
.. banner::
   :variant: cta-banner
```

this is intentional? it's fine if it is, just want to check!

Reply:
Nope, removed it!